validate_bancos.py

```
@@ -0,0 +1,202 @@
+==================
+Valida o arquivo data/bancos.json contra o JSON Schema definido em
+bancos.schema.json.
+
+Uso:
+    python validate_bancos.py [--data PATH] [--schema PATH] [--strict]
+
+Dependências:
+    pip install jsonschema
+
+Retorna:
+    0  – validação OK
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)